src/components/PredictionGame.tsx

```
@@ -42,133 +53,164 @@ interface PredictionGameProps {
+    );
+  const isAdmin = session.user.isAdmin;
+
+  return (
+    <div className="mt-6 flex flex-col gap-6 rounded-xl border border-gray-700 bg-gray-900/50 p-4 shadow-xl backdrop-blur-sm sm:gap-8 sm:p-6">
+      <div
+        className={cn(
+          "flex flex-col gap-2 pb-4",
+          isAdmin ? "cursor-pointer select-none" : "border-b border-gray-700"
+        )}
+        onClick={() => isAdmin && setIsAdminCollapsed(!isAdminCollapsed)}
+      >
```

Biome (2.4.7)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)